packages/core/src/components/Table/stories/TableSamples/Section.tsx

```diff
@@ -0,0 +1,234 @@
+import { useState, useCallback, useMemo } from "react";
+import range from "lodash/range";
+import {
+  HvTableContainer,
+  HvTable,
+  HvTableBody,
+  HvTableHead,
+  HvTableHeader,
+  HvTableRow,
+  HvTableCell,
+  HvPagination,
+  HvTypography,
+  HvBulkActions,
+  HvActionsGeneric,
+  HvSection,
+  useHvPagination,
+  useHvRowSelection,
+  useHvBulkActions,
+  useHvData,
+  theme,
+  tableContainerClasses,
+  tableCellClasses,
+} from "@hitachivantara/uikit-react-core";
+import {
+  Delete,
+  Duplicate,
+  Lock,
+  Preview,
+} from "@hitachivantara/uikit-react-icons";
+
+import { css } from "@emotion/css";
+
+import { AssetEvent, getColumns, makeData } from "../storiesUtils";
+
+const classes = {
+  section: css({
+    padding: 0,
+  }),
+  sectionHeader: css({
+    padding: theme.spacing("xs", "xs", "xs", "sm"),
+    borderBottom: `1px solid ${theme.colors.atmo3}`,
+  }),
+  sectionActions: css({
+    right: theme.space.sm,
+  }),
+  pagination: css({
+    margin: 0,
+    backgroundColor: theme.colors.atmo2,
+    padding: theme.space.xs,
+    borderRadius: `0 ${theme.radii.round} ${theme.radii.round} 0`,
```

Comment:
The left bottom border radius is missing. It should be: `0 0 ${theme.radii.round} ${theme.radii.round}`

Reply:
fixed